apps/dotcom/src/app/pages/faq/FAQ.tsx

```diff
@@ -0,0 +1,72 @@
+import React, {
+  useCallback,
+  useEffect,
+  useImperativeHandle,
+  useMemo,
+} from 'react';
+import { Helmet } from 'react-helmet';
+import { Container, Box, Link } from '@material-ui/core';
+import Hero from '../../components/Hero';
+import { ReactComponent as SVG } from './faq.svg';
+import { Link as RouterLink } from 'react-router-dom';
+import Accordion from './Accordion';
+
+export default function FAQ() {
+  // this is the list of FAQs
+  const frequently_asked_questions = [
+    {
```

Comment:
🔧 Can we move these {title, body} constants out of this file? Maybe to a different file?